A32NX/en-US.locPak

```diff
@@ -8,7 +8,8 @@
       "AIRCRAFT.UI_VARIATION": "Airbus",
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "Adjust selected heading to the left (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_RIGHT": "Adjust selected heading to the right  (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!°)",
-      "COCKPIT.TOOLTIPS.CHRONO_RST": "Reset chronometer"
+      "COCKPIT.TOOLTIPS.CHRONO_RST": "Reset chronometer",
+      "COCKPIT.TOOLTIPS.LIGHTS_NOSMOKE_AUTO": "Set \"no smoke\" prompt light to AUTO mode"
```

Comment:
I think you can leave this out until we actually get it working, else it might be confusing and future people might think that it used to work and was then broken

Reply:
Didn't find any string link to chronometer in the generic tooltips.